src/main/java/edu/ie3/datamodel/io/naming/timeseries/LoadProfileMetaInformation.java

```diff
@@ -37,18 +32,11 @@ public boolean equals(Object o) {
 
   @Override
   public int hashCode() {
-    return Objects.hash(super.hashCode(), profile);
+    return Objects.hash(profile);
```

Comment:
Wouldn't do this, especially because the equals method still relies on the super class